src/components/Emergency.jsx

```diff
@@ -16,15 +16,21 @@ class Emergency extends Component {
 
 	render() {
 		return (
-			<div>
-				<h1 className="ui centered">Call your nearest emergency department.</h1>
-				<p>They will ask more questions to assess you further. You will be brought to the COVID-19 Isolation
-					Area for admission or the Resuscitation Area of the hospital if you don’t want to be admitted.</p>
-				<p>[This part is under construction, please contact DOH directly below.]</p>
-				<Button onClick={this.contactDetails} className="btn btn-primary btn-lg btn-block" role="button">COVID-19
-					Emergency Contact Details</Button>
-				<Button onClick={this.next} className="btn btn-primary btn-lg btn-block" role="button">Restart
+			<div className="col-sm-12 col-md-10 offset-md-1">
+				<h1 className="text-center mt-5 mb-4">Call your nearest emergency department.</h1>
+				<br/>
+				<p>
+					They will ask more questions to assess you further. You will be brought to the COVID-19 Isolation
+					Area for admission or the Resuscitation Area of the hospital if you don’t want to be admitted.
+				</p>
+				<p>
```

Comment:
conflict ata? I've updated the left already.

Reply:
oh. wait. lemme check